pkg/types/nutanix/machinepool.go

```diff
@@ -0,0 +1,57 @@
+package nutanix
+
+// MachinePool stores the configuration for a machine pool installed
+// on Nutanix.
+type MachinePool struct {
+	// NumCPUs is the total number of virtual processor cores to assign a vm.
+	//
+	// +optional
+	NumCPUs int32 `json:"cpus"`
+
+	// NumCoresPerSocket is the number of cores per socket in a vm. The number
+	// of vCPUs on the vm will be NumCPUs/NumCoresPerSocket.
+	//
+	// +optional
+	NumCoresPerSocket int32 `json:"coresPerSocket"`
+
+	// Memory is the size of a VM's memory in MB.
+	//
+	// +optional
+	MemoryMiB int64 `json:"memoryMB"`
```

Comment:
Is this MiB or MB? It is not consistent.

Reply:
Good catch. It is MiB.